src/cephadm/cephadm

```diff
@@ -2184,7 +2184,10 @@ def command_bootstrap():
                               '--allow-overwrite to overwrite' % f)
         dirname = os.path.dirname(f)
         if dirname and not os.path.exists(dirname):
-            raise Error('%s directory %s does not exist' % (f, dirname))
+            if dirname == "/etc/ceph":
```

Comment:
```suggestion
            if dirname == ETC_DIR:
```

Create a constant for this and also use it as the default value for the `--output-dir` arg?

Reply:
Hm. if you _really_ want to change the default locations for ceph's config to something different, then having an alias `ETC_CEPH_DIR` just in `cephadm` doesn't really make any difference. Instead you'd need to look for all occurrences of `/etc/ceph` ecosystem-wide (not just the whole ceph.git) E.g. https://github.com/search?q=%22%2Fetc%2Fceph%22 

Is this really a valid use case?

Reply:
+1 for not being able to change the default value here.

Reply:
I'm not suggesting we change the default location.

Something like this:
https://github.com/ceph/ceph/blob/b62e0c262b43ccf224ec627004898f571db0133d/src/cephadm/cephadm#L8

... to make it more obvious where it's used ... instead of a hard-coded `/etc/ceph` string